test/e2e/package.json

```diff
@@ -19,57 +19,51 @@
 		"decryptconfig": "openssl enc -md sha1 -aes-256-cbc -d -pass env:CONFIG_KEY -in ./config/encrypted.enc -out ./config/local-$NODE_CONFIG_ENV.json"
 	},
 	"dependencies": {
-		"@babel/core": "7.2.2",
-		"@babel/preset-env": "7.3.1",
-		"@babel/register": "7.0.0",
-		"@babel/runtime": "7.3.1",
-		"asana-phrase": "0.0.8",
+		"@babel/core": "^7.2.2",
+		"@babel/preset-env": "^7.3.1",
+		"@babel/register": "^7.0.0",
+		"@babel/runtime": "^7.3.1",
+		"asana-phrase": "^0.0.8",
 		"cached-path-relative": ">=1.0.2",
 		"check-npm-client": "^1.1.1",
 		"chromedriver": "^80.0.0",
 		"concurrently": "^3.6.1",
-		"config": "1.28.0",
+		"config": "^1.28.0",
 		"cryptiles": ">=4.1.2",
 		"esformatter-braces": "^1.2.1",
 		"esformatter-collapse-objects-a8c": "^0.1.0",
 		"esformatter-dot-notation": "^1.3.1",
 		"esformatter-quotes": "^1.0.3",
 		"esformatter-semicolons": "^1.1.2",
 		"esformatter-special-bangs": "^1.0.1",
-		"esm": "3.2.10",
-		"ffmpeg-static": "2.4.0",
-		"fs-extra": "0.22.1",
+		"esm": "^3.2.10",
+		"ffmpeg-static": "^2.4.0",
+		"fs-extra": "^0.22.1",
 		"grunt": "^1.0.3",
 		"grunt-cli": "^1.2.0",
 		"grunt-concurrent": "^2.3.0",
 		"grunt-shell": "^1.1.2",
-		"junit-viewer": "4.9.6",
+		"junit-viewer": "^4.9.6",
 		"lodash": "^4.13.1",
-		"mailosaur": "4.0.0",
+		"mailosaur": "^4.0.0",
 		"mocha": "^5.2.0",
 		"mocha-steps": "^1.1.0",
-		"node-slack-upload": "1.2.1",
-		"png-itxt": "1.3.0",
+		"node-slack-upload": "^1.2.1",
+		"png-itxt": "^1.3.0",
 		"prettier": "npm:wp-prettier@1.19.1",
 		"push-receiver": "^2.0.0",
 		"request-promise": "^4.2.2",
-		"sanitize-filename": "1.6.0",
+		"sanitize-filename": "^1.6.0",
 		"saucelabs": "^1.5.0",
-		"selenium-webdriver": "4.0.0-alpha.4",
+		"selenium-webdriver": "^4.0.0-alpha.4",
 		"slack-notify": "^0.1.7",
 		"speakeasy": "^2.0.0",
 		"spec-xunit-reporter": "^0.0.1",
-		"testarmada-magellan": "11.0.1",
+		"testarmada-magellan": "^11.0.1",
 		"testarmada-magellan-local-executor": "^2.0.0",
 		"testarmada-magellan-mocha-plugin": "github:Automattic/magellan-mocha-plugin#b731cdf",
 		"xml2json-command": "^0.0.3",
 		"xmpp.js": "^0.3.0",
 		"xunit-viewer": "^5.1.8"
-	},
-	"devDependencies": {
-		"babel-eslint": "10.0.1",
-		"circleci": "^0.3.3",
-		"eslint": "5.12.0",
-		"husky": "^0.15.0-rc.8"
```

Comment:
`circleci` moved to the root to follow the convention of not having devDependencies in child packages.
The other packages were already there.